pkg/e2e/machinehealthcheck/machinehealthcheck.go

```diff
@@ -113,8 +113,10 @@ var _ = Describe("[Feature:MachineHealthCheck] MachineHealthCheck controller", f
 		})
 	})
 
-	It("should delete unhealthy machine", func() {
-		stopKubeletAndValidateMachineDeletion(workerNode, workerMachine, 6*time.Minute)
+	Context("without node-unhealthy-conditions configmap", func() {
+		It("should delete unhealthy machine", func() {
+			stopKubeletAndValidateMachineDeletion(workerNode, workerMachine, 7*time.Minute)
```

Comment:
I don't see how putting the `It` clause under `Context` help with to fix this test. It just might be the timeout that got increased that helped. We might bump the timeout to even 10 minutes. Though, that will only hide the real issue. This might also be a bug in the HC itself.

Reply:
putting IT under context is just adding correct description about test case. 

5 minutes is timeout in the configmap  so machine deletion get triggered after 5 minutes. With keeping test case timeout as 7 min, effectively it is waiting for 2 mins. If we still see I failures, we can debug where exactly its taking delayed. For now, i would prefer to try out 7 min timeout.